tests/integration/security/testdata/authz/v1beta1-egress-gateway.yaml.tmpl

```diff
@@ -26,23 +26,23 @@ spec:
             hosts: ["www.company.com"]
     - to: # checks only a call 443 over istio mutual without JWT
       - operation:
-          hosts: ["a-only.com"]
+          hosts: ["{{ .a }}-only.com"]
```

Comment:
I think we don't need to change the template policy if it's not used by the vm at all (because gateway can not be deployed as vm for now).

Reply:
by the way I feel it makes more sense to replace `a` or `c` with `vm` (in the future both should be replaced) since the more common case is that the workload (e.g. `vm`) is accessing some out-of-mesh service through egress gateway, not the reverse. but I think this is up to you.

Reply:
Good point. I have reverted changes to b, and switch between a and vm for the test. 
Now the flow is `a/vm, c --> egress gateway --> b`.
The templates also keeps `b` in host name instead of parameter `{{ .b }}`.

Reply:
@yangminzhu please take a look at the egress gateway test. Thanks a lot!